src/server/game/Handlers/DuelHandler.cpp

```diff
@@ -35,9 +35,14 @@ void WorldSession::HandleDuelAcceptedOpcode(WorldPacket& recvPacket)
     sLog->outStaticDebug("Player 2 is: %u (%s)", plTarget->GetGUIDLow(), plTarget->GetName().c_str());
 #endif
 
+   if (player->duel->isUnderDuel || plTarget->duel->isUnderDuel)
```

Comment:
I'm wondering where we check that `plTarget->duel` is not null, I only see the check for `GetPlayer()->duel` above

if `plTarget->duel` is null, with this code it will crash

Reply:
https://github.com/azerothcore/azerothcore-wotlk/blob/38206092f2666a2a7496757bdb6fa47c5e94b62b/src/server/game/Handlers/DuelHandler.cpp#L29

Reply:
https://github.com/azerothcore/azerothcore-wotlk/blob/38206092f2666a2a7496757bdb6fa47c5e94b62b/src/server/game/Entities/Player/Player.cpp#L7906

In this file that can be deleted!

Thats need a big test and try to get a crash !

Reply:
or maybe need to use :
```if ((player->duel && player->duel->isUnderDuel) || (plTarget->duel && plTarget->duel->isUnderDuel))```